File: C++/060. Bissexto/Bissexto.cpp
#include <iostream>
int main()
{
    int A;
    std::cin >> A;
    if (A % 400 == 0) {
        std::cout << "ANO BISSEXTO" << std::endl;
        return 0;
    }
    else if (A % 100 == 0) {
        std::cout << "ANO NAO BISSEXTO" << std::endl;
        return 0;
    }
    else if (A % 4 == 0) {
        std::cout << "ANO BISSEXTO" << std::endl;
        return 0;
    }
    else {
        std::cout << "ANO NAO BISSEXTO" << std::endl;
        return 0;
    }
   return 0;
}

File: Java/060. Bissexto/Bissexto.java
import java.util.Scanner;
public class Bissexto
{
    public static void main(String[] args) 
    {
        Scanner scan = new Scanner(System.in);
        int ano = scan.nextInt();
        if (ano % 400 == 0)
        {
            System.out.println("ANO BISSEXTO");
        }
        else if (ano % 100 == 0)
        {
            System.out.println("ANO NAO BISSEXTO");
        }
        else if (ano % 4 == 0)
        {
            System.out.println("ANO BISSEXTO");
        }
        else
        {
            System.out.println("ANO NAO BISSEXTO");
        }
        scan.close();   
    }
}